sonar-plugin-api/src/main/java/org/sonar/api/test/CoverageBlock.java

```diff
@@ -21,6 +21,10 @@
 
 import java.util.List;
 
+/**
+ * @deprecated since 5.6 feature will be removed soon
```

Comment:
The message is not clear enough. I suggest `@deprecated since 5.6. Feature will be removed without any alternatives`.